pkgs/tools/typesetting/tex/opendetex/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, flex
+, testers
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "opendetex";
+  version = "2.8.9";
+
+  src = fetchFromGitHub {
+    owner = "pkubowicz";
+    repo = "opendetex";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-8mwXsJImi8jESR87VA9NsUIvH52JJbbrjzpUdKPI+ls=";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace "install -c -m 775 -g staff" "install -c -m 775" \
+      --replace "/usr/local" "$out"
```

To make it build on Darwin:
```suggestion
      --replace "LEXLIB	= -ll" "LEXLIB	= -lfl" \
      --replace "/usr/local" "$out"
```
with a tab before the `=`, or the patch won't take.